### PR TITLE
Test suggestions

### PR DESCRIPTION
**Pull request by millerdev**
I moved some things around, removed some unneeded things, documented the setup, etc. Have a look over each commit and make a note of anything that doesn't make sense.

Note that there is still more work to do here. The main thing is all [TestScripts](https://github.com/dimagi/dimagi-qa/tree/a433ee23f9b704fbbe00413167ff61759260ddeb/TestScripts) files need to be renamed to `test_....py` as was done in https://github.com/dimagi/dimagi-qa/pull/5/commits/a433ee23f9b704fbbe00413167ff61759260ddeb

I also strongly recommend to remove all `time.sleep()` calls where possible. Most of the time they are not necessary.

### CODE REVIEW

**Review by kiran-tripathy — COMMENTED** [review]
Since we removed the Html Test Runner, how would the test results be generated as a report?
@millerdev

**Review by kiran-tripathy — COMMENTED** [review]
Acknowledging that you have already mentioned that there is still stuff left to do; but would this setup be a manual process?

File: Pages/loginPage.py [hunk 1]
```diff
@@ -26,8 +25,6 @@ def click_submit(self):
...
     def accept_alert(self):
         try:
-            WebDriverWait(self.driver, 10).until(
-                ec.presence_of_element_located((By.ID, self.alert_button_accept))).click()
-            time.sleep(7)
-        except Exception as e:
-            print(e)
+            self.driver.find_element_by_id(self.alert_button_accept).click()
+        except NoSuchElementException:
+            pass  # ignore if alert not on page
```
Comment (millerdev) [inline Pages/loginPage.py:30]: @kiran-tripathy let me know if this does not work on your system. We may be able to take a different approach if it does not.

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[tool:pytest]
+addopts = --self-contained-html --html=report.html
```
Comment (kiran-tripathy) [inline setup.cfg:2]: There are implementation(s) pending for the report, yeah?
Or should the reports be generating already?

File: Pages/loginPage.py (hunk 1 above)
Comment (millerdev) [reply Pages/loginPage.py:30]: It should be called there, I added it:

https://github.com/dimagi/dimagi-qa/blob/e330af3afb62bcaa49f6061d357ac06d03cfd778/TestBase/environmentSetupPage.py#L25

File: setup.cfg (hunk 2 above)
Comment (millerdev) [reply setup.cfg:2]: The report should be generated when you run `pytest`. You'll find a file called `report.html`

Comment (kiran-tripathy) [reply setup.cfg:2]: Yes, I just checked and this is working.

File: Pages/loginPage.py (hunk 1 above)
Comment (kiran-tripathy) [reply Pages/loginPage.py]: My bad, I hadn't pulled your changes. This was still failing test cases, si I added 
https://github.com/dimagi/dimagi-qa/commit/978e2d7f1ad3564378cfecd69f71c2f72bccfbd5

**Pull request merged**